### PR TITLE
chore: enable linting and arrow functions

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -193,37 +194,36 @@ export class Translate extends common.Service {
     input = arrify(input);
 
     this.request(
-      {
-        method: 'POST',
-        uri: '/detect',
-        json: {
-          q: input,
+        {
```
Comment (stephenplusplus) [inline src/index.ts]: This went to a 4-space indent. Did a tool do that automatically?

File: src/index.ts [hunk 2]
```diff
@@ -22,19 +22,20 @@ import * as extend from 'extend';
...
 
 /**
  * @typedef {object} ClientConfig
  * @property {string} [projectId] The project ID from the Google Developer's
  *     Console, e.g. 'grape-spaceship-123'. We will also check the environment
  *     variable `GCLOUD_PROJECT` for your project ID. If your app is running in
- *     an environment which supports {@link https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application Application Default Credentials},
- *     your project ID will be detected automatically.
+ *     an environment which supports {@link
+ * https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application
```
Comment (stephenplusplus) [inline src/index.ts]: In this and other places, the doc formatting ends up un-indented when it reaches the next line, where we normally have 4 spaces. Is that done by a formatting tool?

File: src/index.ts (hunk 1 above)
Comment (JustinBeckwith) [reply src/index.ts]: Yes, this was `clang-format`

File: src/index.ts (hunk 2 above)
Comment (JustinBeckwith) [reply src/index.ts]: Yes, this was `clang-format`

File: src/index.ts (hunk 1 above)
Comment (stephenplusplus) [reply src/index.ts]: Follow-up! Is that what we want / is it configurable / who cares?

Comment (JustinBeckwith) [reply src/index.ts]: For now, yeah it's want we want.  In fullness of time, we want to switch to prettier.  Discussion here:
https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2842

**Review by stephenplusplus — APPROVED** [review]

**Pull request merged**